configs/trainer.yaml

```diff
@@ -1,5 +1,5 @@
 trainer_kwargs:
-  max_epochs: 100
+  max_epochs: 10
```

Comment:
is this enough for convergence?

Reply:
Yes - looking at the wandb plots val_loss seems to plateau after about 4-6 epochs at most. Could lower this now or leave for #26?

Reply:
Cool. Happy to leave for 26